engine-core/src/main/java/io/nosqlbench/engine/core/lifecycle/ScenarioResult.java

```diff
@@ -170,8 +156,7 @@ public void reportCountsTo(PrintStream printStream) {
                 }
             } else if (v instanceof Gauge) {
                 Object value = ((Gauge) v).getValue();
-                if (value != null && value instanceof Number) {
-                    Number n = (Number) value;
+                if (value != null && value instanceof Number n) {
```

Comment:
You can remove the null check on this one.

Reply:
done and pushed